amqp-postgres/amqp_postgres/tests/tests.py

```diff
@@ -0,0 +1,221 @@
+            error='',
+            parameters={},
+            is_system_workflow=False
+        )
+        new_execution.creator = admin_user
+        new_execution.tenant = default_tenant
+
+        self.sm.put(new_execution)
+
+    def _get_db_element(self, model):
+        items = self.sm.list(model)
+        self.assertEqual(len(items), 1)
+        return items[0]
+
+    def _assert_timestamp(self, elem):
+        timestamp = date_parser.parse(elem.timestamp)
+        reported_timestamp = date_parser.parse(elem.reported_timestamp)
+
+        # timestamp comes from `postgres_publisher` when creating the new
+        # element, while `reported_timestamp` comes from the message object,
+        # which should be created beforehand
+        self.assertGreaterEqual(timestamp, reported_timestamp)
+
+    def _assert_log(self, log, db_log):
+        self._assert_timestamp(db_log)
+
+        self.assertEqual(db_log.message, log['message']['text'])
+        self.assertEqual(db_log.message_code, None)
+        self.assertEqual(db_log.logger, log['logger'])
+        self.assertEqual(db_log.level, log['level'])
+        self.assertEqual(db_log.operation, log['context']['operation'])
+        self.assertEqual(db_log.node_id, log['context']['node_id'])
+        self.assertEqual(db_log.execution.id, log['context']['execution_id'])
+        self.assertEqual(db_log.creator.id, 0)
+        self.assertEqual(db_log.tenant.id, 0)
+        self.assertEqual(db_log.reported_timestamp, log['timestamp'])
+        self.assertEqual(db_log.private_resource, False)
+        self.assertEqual(db_log.visibility, VisibilityState.TENANT)
+
+    def _assert_event(self, event, db_event):
+        self._assert_timestamp(db_event)
+
+        self.assertEqual(db_event.message, event['message']['text'])
+        self.assertEqual(db_event.message_code, None)
+        self.assertEqual(db_event.event_type, event['event_type'])
+        self.assertEqual(db_event.error_causes, None)
+        self.assertEqual(db_event.operation, None)
+        self.assertEqual(db_event.node_id, None)
+        self.assertEqual(db_event.execution.id,
+                         event['context']['execution_id'])
+        self.assertEqual(db_event.creator.id, 0)
+        self.assertEqual(db_event.tenant.id, 0)
+        self.assertEqual(db_event.reported_timestamp, event['timestamp'])
+        self.assertEqual(db_event.private_resource, False)
+        self.assertEqual(db_event.visibility, VisibilityState.TENANT)
+
+    @staticmethod
+    def _get_log(execution_id):
+        return {
+            'context': {
+                'blueprint_id': 'bp',
+                'deployment_id': 'dep',
+                'execution_id': execution_id,
+                'node_id': 'vm_7j36my',
+                'node_name': 'vm',
+                'operation': 'cloudify.interfaces.cloudify_agent.create',
+                'plugin': 'agent',
+                'task_id': 'a13973d5-3866-4054-baa1-479e242fff75',
```

Comment:
this is not asserted on, right? maybe it shouldnt be here at all, or do you prefer to keep for reference?
(ditto "logger")